BasiliskII/src/Unix/main_unix.cpp

```diff
@@ -448,6 +455,19 @@ int main(int argc, char **argv)
 			vde_sock = argv[i];
 			argv[i] = NULL;
 		}
+		
+#if defined(__APPLE__) && defined(__MACH__)
```

Comment:
Should this be __MACOSX__ instead for consistency with other files?

Reply:
Done